### ISSUE TITLE
[NTR/gene] obsolete genetic skin disease

### ISSUE THREAD

**Issue by pnrobinson** [monarch-initiative/mondo#5682]
**Preferred gene-related syndrome label**

It seems to be that genodermatosis (lay synonym genetic skin disease) is a good term
@ahamosh  -- thoughts?

The term was made obsolete, but I think we want to keep it!

obsolete genetic skin disease

[obsolete genetic skin disease](https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0024255)

**Comment by ahamosh** [comment]
Why would you obsolete genodermatosis?

**Comment by nicolevasilevsky** [comment]
at one point we decided we didn't want to include genetic grouping classes because it was too difficult to keep the classifications current and up to date. But it sounds like we do want to have these terms (we discussed on the curation call and we all agreed to add these terms back, as needed).

@pnrobinson plans to work on this ticket. Note, we should not reinstate an obsoleted term, rather we should create a new term with a new ID:
https://mondo.readthedocs.io/en/latest/editors-guide/merging-and-obsoleting/#restoring-an-obsolete-ontology-term

**Comment by pnrobinson** [comment]
@nicolevasilevsky @sabrinatoro can we add this term? If we could do a session together maybe you could show me how to make a PR then I could do things like this myself!

**Comment by nicolevasilevsky** [comment]
Hi Peter, yes, we can add it. I can go ahead and take care of it and we'd be happy to have an editing session with you to show you how to make PRs. We're meant to have a medical experts curation call on Thursday, we can plan to do it then.

**Comment by ahamosh** [comment]
I agree with un-obsoleting (is that a word) genodermatosis.  I was supposed to be in Cape Town this week, but my dad got sick, so I am in town.  I might be able to make the curation call on Thursday.
